Netkan/Services/ModuleService.cs

```diff
@@ -116,7 +116,13 @@ private static AvcVersion GetInternalAvc(CkanModule module, ZipFile zipfile, str
 
                 for (var i = 0; i < files.Count; i++)
                 {
-                    if (files[i].Name == internalFilePath)
+                    Log.DebugFormat("Testing file '{0}' against path '{1}'", files[i].Name,internalFilePath);
+                    if (files[i].Name == internalFilePath) 
+                    {
+                        remoteIndex = i;
+                        break;
+                    }
+                    else if (Regex.IsMatch(files[i].Name, internalFilePath))
```

Comment:
the "?"  declares a nullable type, and means that the type before it may have a null value.